fern/docs/pages/overview/quickstart.mdx

```diff
@@ -32,6 +32,22 @@ being used
 http://localhost:8001/
 ```
 
+You like one-liners, have python3.11 installed, and you are running a UNIX (MacOS & Linux) system?
```

Comment:
This content mostly repeats the previous one. I'd suggest to delete the previous one in this section as just go with this new one. Something like:

> The steps in [Installation](https://privategpt-preview-80122a6c-f00a-41eb-948a-5742f2e48d6b.docs.buildwithfern.com/installation) section are better explained and cover more setup scenarios (MacOS, Windows, Linux). But if you like one-liners, have python3.11 installed, and you are running a UNIX (MacOS & Linux) system, you can get up and running on CPU in few lines:

That way we remove the first code block.